.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Global ownership
+* @genmcp/maintainers
```

```suggestion
* @genmcp/maintainers

```

nit: enter empty line at end